searx/engines/google_images.py

```diff
@@ -95,8 +91,7 @@ def response(resp):
     json_start = resp.text.find('{"ischj":')
     json_data = loads(resp.text[json_start:])
 
-    for item in json_data["ischj"]["metadata"]:
-
+    for item in json_data["ischj"].get("metadata", []):
```

Comment:
@jazzzooo this is the only line that is needed to fix the bug / please revert unneeded reformatted code in this PR / thanks!